.github/workflows/pnpm-autoupdate.yml

```diff
@@ -0,0 +1,41 @@
+name: PNPM auto update packages
+
+on: 
+  pull_request:
+    branches: [devel]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: ['18']
+    name: pnpm packages update
+    steps:
+      - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 7
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+          cache: 'pnpm'
+          cache-dependency-path: swift_browser_ui_frontend/pnpm-lock.yaml
+      - name: Install dependencies
+        run: |
+          pushd swift_browser_ui_frontend
+          pnpm update
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          commit-message: Update pnpm packages
+          title: PNPM package updates
+          body: Automated PR with PNPM updates
+          branch: update/pnpm-updates
```

Comment:
maybe we can create a new branch, so adding a random nb at the end would do.
They will get deleted on merge either way.

Reply:
If we do that, we may end up with multiple pull requests. When the packages are outdated, each new pull request would create a new pnpm update pull request. If we use the same branch name (current version), we get only one PR.

More details in the github action documentation that creates PRs https://github.com/peter-evans/create-pull-request#action-behaviour